qemu/tests/cfg/virtio_serial_file_transfer.cfg

```diff
@@ -16,3 +16,31 @@
     Windows:
         guest_script_folder = C:\
         clean_cmd = del /f /q
```

Comment:
" file_sender = both " is used now,
Can we separate it to file_sender=guest and file_sender=host,
and run sub-test when guest -> host and host -> guest.

Regards,
Suqin